scripts/deploy.sh

```diff
@@ -40,3 +40,22 @@ npm run build
 echo "$CDN_PRIVATE_KEY" > id_rsa
 ./scripts/sftpCommands.sh ${VERSION} | \
   sftp -i ./id_rsa -oHostKeyAlgorithms=+ssh-dss -b - sshacs@dxresources.ssh.upload.akamai.com:/prod/alloy
+
+# verify the files are available on the cdn
+ALLOY_MIN_JS="https://cdn1.adoberesources.net/alloy/${VERSION}/alloy.min.js"
+STATUS_CODE=$(curl -o /dev/null -Isw '%{http_code}\n' $ALLOY_MIN_JS)
+if [ $STATUS_CODE != "200" ]; then
+  echo "curl request to $ALLOY_MIN_JS returned status code $STATUS_CODE" 1>&2
+  exit 1
+fi
+
+ALLOY_JS="https://cdn1.adoberesources.net/alloy/${VERSION}/alloy.js"
+STATUS_CODE=$(curl -o /dev/null -Isw '%{http_code}\n' "${ALLOY_JS}")
```

Comment:
Is there an important reason why `"${ALLOY_JS}"` has quotes but `$ALLOY_MIN_JS` doesn't?

Looks good, though! 🚢 🇮🇹